src/components/valences-test/ValencesTest.test.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from "react";
+import { Router } from "react-router-dom";
+import { render, screen, within, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { createMemoryHistory } from "history";
+
+import ElementManager from "../../ElementManager";
+import ValencesTest from "./ValencesTest";
+
+const history = createMemoryHistory();
+
+const setup = () => {
+  ElementManager.loadElements();
+  return render(
+    <Router history={history}>
+      <ValencesTest />
+    </Router>
+  );
+};
+
+test("Should render a question and four answers", () => {
+  const { container } = setup();
+  const questionCard = container.querySelector(".question-card") as HTMLElement;
+  expect(questionCard).toBeInTheDocument();
+  const buttons = within(questionCard).getAllByRole("button");
+  expect(buttons).toHaveLength(4);
+});
+
+test("should display a new question when clicking on the correct answer", () => {
```

Comment:
I would like to synchronize with the common convention we use

Reply:
your guess is as good as mine, I haven't look in the repo for the whole week,, I will probably check tomorrow 🤞

Reply:
I think this sort of thing is a work in progress... it's a good idea to perhaps establish a convention.  I guess it will probably depend on the project we are raiding at the time.  Personally, I'm not that worried about what is used (i prefer the `describe` `test` over `it` myself), I just expect there to be a clear explanation/description given about what is being tested.